### PR TITLE
Create 2018-07-24-wechaty-installation-in-windows-10.md

### PR DESCRIPTION
**Pull request by kaffa**
This pull request has been automatically generated by prose.io.

### CODE REVIEW

File: _posts/2018-07-24-wechaty-installation-in-windows-10.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
+---
+published: false
+---
```
Comment (lijiarui) [inline _posts/2018-07-24-wechaty-installation-in-windows-10.md]: Please change to the following code:
```
---
 layout: post
 title: "'在 Windows 10 下安装 Wechaty"
 date: 2018-07-24 09:00 +0800
 author: kaffa
 ---

> kaffa, Ex-Tencent, Ex-CMB, doing personal investment now.

(Or you can try other introduction you like:) )
```

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Comment by lijiarui** [comment]
Thanks for your pr, please change the title as I request, thanks again!

File: _posts/2018-07-24-wechaty-installation-in-windows-10.md [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+## 在 Windows 10 下安装 Wechaty
+
+感觉 @zixia 让我在此分享我在 Windows 10 Home 平台安装 WeChaty 的经历，希望我对安装过程的记录和分析能帮助在此平台下安装 Wechaty 的人。
```
Comment (huan) [inline _posts/2018-07-24-wechaty-installation-in-windows-10.md]: 感觉 -> 感谢

File: _posts/2018-07-24-wechaty-installation-in-windows-10.md [hunk 3]
```diff
@@ -0,0 +1,359 @@
+---
+published: false
```
Comment (huan) [inline _posts/2018-07-24-wechaty-installation-in-windows-10.md]: Please add all required headers, or this blog will not be published right.

File: _posts/2018-07-24-wechaty-installation-in-windows-10.md [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+感觉 @zixia 让我在此分享我在 Windows 10 Home 平台安装 WeChaty 的经历，希望我对安装过程的记录和分析能帮助在此平台下安装 Wechaty 的人。
+
+
+### 介绍
+
+[Wechaty](https://github.com/Chatie/wechaty) 是一个对话机器人软件开发包，它可以帮你用 6 行 JavaScript 代码实现一个微信个人账号。由于使用 node 技术，它支持 Linux、Windows、macOS 平台和 Docker 容器。它的安装过程和其它 node 包类似，首先 ```git clone``` 源代码，再使用 node 的 npm 命令 ```npm install & npm start``` 实现开箱即用，同时官方也提供了入门案例：Wechaty getting started。而在国内，实际安装过程中可能会因某些软件依赖的原因遇到问题，本文记录了安装过程中可能会出现的错误及解决办法，下面描述详细的步骤和说明：
+
+### 概念
+
+[Wechaty](https://github.com/Chatie/wechaty)
```
Comment (huan) [inline _posts/2018-07-24-wechaty-installation-in-windows-10.md]: Changed to 

```
* 项目地址 [Wechaty](https://github.com/Chatie/wechaty)
```

Would be better for display.

File: _posts/2018-07-24-wechaty-installation-in-windows-10.md [hunk 5]
```diff
@@ -0,0 +1,359 @@
...
+8. 至此，应该可以顺利运行起步项目:
+```
+npm install & npm start
+```
+
+### 总结
+
+由于已成功启动项目，所以我并没有在 Windows 7 系统 或 32 位机上进行配置，主要问题应该差不多。 解决办法基本上一一查阅上述出现错误的软件的文档，应该可以独立解决。
+
+当然，如果你遇到了任何其他问题，也欢迎 我的Github 上联系我。
```
Comment (huan) [inline _posts/2018-07-24-wechaty-installation-in-windows-10.md]: Suggest adding your Github link to this line. 

And also it is good to add your original blog url at here.

**Review by huan — CHANGES_REQUESTED** [review]
Thank you very much for the post PR!

Could you please follow my reviews and fix the article meta header so that we can be able to merge it?

File: _posts/2018-07-24-wechaty-installation-in-windows-10.md [hunk 6]
```diff
@@ -0,0 +1,368 @@
...
+title: "在 Windows 10 Home 下安装 Wechaty"
+date: 2018-07-24 09:00 +0800
+author: kaffa
+---
+
+> [@kaffa](https://gitbuh.com/kaffa) a life-long learner / a beginner investor / a coffee mocha lover. My blog is [https://kaffa.im/](https://kaffa.im/) 
+
+## 在 Windows 10 下安装 Wechaty
+
+![在 Windows 10 下安装 Wechaty](https://github.com/kaffa/blog/blob/master/download/2018/wechaty-installation-in-windows.png)
```
Comment (huan) [inline _posts/2018-07-24-wechaty-installation-in-windows-10.md:12]: It seems that you had forgotten to include the image file in this PR.

Comment (kaffa) [reply _posts/2018-07-24-wechaty-installation-in-windows-10.md:12]: Sorry, it's my first time to send a PR and I couldn't figure out how to include the image, so I send a new PR.

Comment (huan) [reply _posts/2018-07-24-wechaty-installation-in-windows-10.md:12]: No problem, send a new PR is ok, and your image is great!

**Comment by huan** [comment]
Thank you very much for your awesome post!

**Pull request merged**